### DISCUSSION TITLE
DataView gridItem Changed Behavior in Angular 17 Upgrade

### DISCUSSION THREAD

**Discussion by andrew-holman** [primefaces/community#722] [PrimeNG Templates] [answered]
After upgrading PrimeNG to 17.1.0 none of my DataView elements work correctly. I use the grid layout so most of them look something like this.
  
```
 <div>
    <p-dataView #prodList [value]="['prod1', 'prod2', 'prod3', 'prod4']" layout="grid">
      <ng-template let-product pTemplate="gridItem">
          <div class="col-12 md:col-4">
              {{ product }}
          </div>
      </ng-template>
    </p-dataView>
 </div>
```

It seems as if the gridItem is now being treated as just a 'grid' since all of the products are being listed in a single element. I was able to fix this by using *ngFor like the documentation shows, but I'd prefer to avoid having to make that change everywhere in my application. Is there a way to get the old behavior for the grid layout?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

This issue happens because we've removed primeflex dependency from dataView and introduced new templating options. DataView will no longer loop the value object, it exposes and lets the user loop through the data. Unfortunately, there isn't a workaround for it.

**Reply by andrew-holman** [reply]
Thanks for the info, is there any place where all changes like this are listed for Angular 17. I just want to confirm nothing else changed for us.
